.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,5 +1,8 @@
-#Tue Jan 24 17:02:29 PST 2012
 eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.6
```

Comment:
I don't think so. This is exactly the reason, I wanted to ignore `.settings` folder from git repo